modules/local/dipcall.nf

```diff
@@ -41,17 +41,17 @@ process DIPCALL {
     def prefix = task.ext.prefix ?: "${meta.id}"
     if(sex == '2') {
```

Comment:
Instead of using an if statement for sex, you could define a variable that is --cs if sex=2 and that its empty if sex=1 (I might be overlooking something here). You can do something like [here](https://github.com/genomic-medicine-sweden/tomte/blob/86de5040d712757d6bebf8d6e9342515cdcdc0de/modules/local/drop_config_runAE.nf#L35C5-L35C70)

Reply:
Ah, this whole "module" is a mess, and I don't think we will keep it. And if kept, it should probably be reworked into a subworkflow. When running the test data, `--cs` should always be used if the test reference genome is large, regardless of sex. If sex == 1, then basically line 61-63 is replaced with 89-95.